src/components/card/card-link/card-link.ts

```diff
@@ -10,7 +10,9 @@ import { SbbCardActionCommonElementMixin } from '../common/index.js';
  *   This is relevant for SEO and screen readers.
  */
 @customElement('sbb-card-link')
-export class SbbCardLinkElement extends SbbCardActionCommonElementMixin(SbbLinkBaseElement) {}
+export class SbbCardLinkElement extends SbbCardActionCommonElementMixin(SbbLinkBaseElement) {
+  protected override actionRole: 'link' | 'button' = 'link';
```

Comment:
```suggestion
  protected override actionRole = 'link';
```

Reply:
Unfortunately the linting gets angry at that :/